src/common/components/repositories-list/index.js

```diff
@@ -5,9 +5,9 @@ import { conf } from '../../helpers/config';
 import { createSnap } from '../../actions/create-snap';
```

Comment:
This component needs a new name, because it doesn't show repositories. It shows snaps.

How does everyone feel about `SnapList`? (cc: @cjwatson @bartaz @earnubs)